apps/dolly-frontend/src/main/js/src/components/ui/button/FrigjoerButton/FrigjoerButton.tsx

```diff
@@ -75,13 +73,7 @@ export const FrigjoerButton = ({
 						<NavButton
 							onClick={() => {
 								closeModal()
-								if (importerteRelatertePersoner) {
```

Comment:
Om vi ikke skal ta høyde for importerte relaterte personer lenger hadde det vært flott om du kunne ryddet opp og fjernet det andre her som er relatert til dette 😎

Reply:
Sløvt å ikke gjøre det med en gang, er ryddet nå!